src/test/runTest.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from "path";
+
+import * as glob from "glob";
+import { runTests } from "@vscode/test-electron";
+
+async function main() {
+  try {
+    // The folder containing the Extension Manifest package.json
+    // Passed to `--extensionDevelopmentPath`
+    const extensionDevelopmentPath = path.resolve(__dirname, "../../");
+
+    // The path to the extension test script
+    // Passed to --extensionTestsPath
+    const extensionTestsPath = path.resolve(__dirname, "./suite/index");
+
+    const vscodePaths = glob.sync(
+      `${extensionDevelopmentPath}/.vscode-test/vscode-*/code`
+    );
+
+    if (vscodePaths.length == 0) {
+      throw new Error("Unable to find vscode executable");
+    }
+
+    const vscodeExecutablePath = vscodePaths[0];
+
+    // Download VS Code, unzip it and run the integration test
```

Comment:
This comment isn't accurate. we have another script to download and unzip VS Code.

Which means the next line only runs tests and therefore `runTests` is sufficient and the comment can be deleted.
```suggestion
```